lib/dashboard/dashboard.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Can you add a sanity check test here? To check that this HTML is valid.

Reply:
added the sanity check in adapter.go, I basically just call html.Parse() and see if any errors get returned